site-manage/site-manage-tool/tool/src/bundle/sitesetupgeneric.properties

```diff
@@ -890,6 +890,8 @@ ediacc.cannotbejoi	= Limit to official course members or to those I add manually
 ediacc.canbejoi     = Allow any {0} user to join the site
 ediacc.vis			= Site Visibility
 ediacc.vis.instr	= Your site can be listed in the Site Browser on the Gateway, allowing non-participants to find it and access its public resources.
+ediacc.userpresence = User Presence Tool
+ediacc.userpresence.instr    = Your site's participants can view other online participants of the site with the presence tool on the bottom right of the site. You can disable this to enable anonymity for the site.
```

Comment:
I'm not sure that saying "You can disable this to enable anonymity for the site." is quite right, it might give a false impression that everyone accessing the site is now anonymous, which is not the case.